_docs/_latest/develop/dev-setup.md

```diff
@@ -185,20 +166,26 @@ following (use the values from your terminal, not "`/SOME/PATH/TO`"):
 
 Before you run Forseti, you need to edit the forseti_conf.yaml file, found in
 `forseti-security/configs/forseti_conf.yaml`. Refer to 
-["Configuring Forseti"]({% link _docs/latest/configure/configuring-forseti.md %}) 
+["Configuring Forseti"]({% link _docs/latest/howto/configure/configuring-forseti.md %}) 
```

Comment:
The troubleshooting steps for Mac/Homebrew should go away if you remove them from this guide.